packages/keyring-eth-hd/src/index.js

```diff
@@ -116,14 +114,14 @@ class HdKeyring {
     this.hdPath = opts.hdPath || hdPathString;
 
     if (opts.mnemonic) {
-      this._initFromMnemonic(opts.mnemonic);
+      await this._initFromMnemonic(opts.mnemonic);
```

Comment:
Despite the title of this PR (`Move seed generation to deserialization`), I understand that the seed generation was already done using the `deserialize` method, though now we are not automatically calling `deserialize` during Keyring construction. Is this correct to say?

Reply:
Yeah, the title is meant to imply that directly calling the deserialization function is the only way to trigger seed generation now, except for `generateRandomMnemonic`